neural_compressor/experimental/nas/dynas.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, conf_fname_or_obj):
         self.macs_predictor = None
         self.latency_predictor = None
         self.results_csv_path = None
+        self.num_workers = 20
```

Comment:
Perhaps we should expose `num_workers` to be configurable via config and assign it here - https://github.com/intel/neural-compressor/blob/7c48cac0ba944098df9a55b0799cd6c41bf3866c/neural_compressor/experimental/nas/basic_nas.py#L70 ?

Reply:
Pushed new commit, this should help.